### PR TITLE
[046]add updateFormStatus

### PR DESCRIPTION
**Pull request by ShirshovAlex**
add updateStatus endpoint

### CODE REVIEW

**Review by OlgaEvdokimova — DISMISSED** [review]

File: src/main/resources/application.yaml [hunk 1]
```diff
@@ -24,7 +24,6 @@ spring:
       - 'classpath:db/changelog/v.0.4/sql-import/import-form-add-column-fm-inship-id.sql'
       - 'classpath:db/changelog/v.0.4/sql-import/import-user-internship.sql'
       - 'classpath:db/changelog/v.0.4/sql-import/import-user-hash-password.sql'
-      - 'classpath:db/changelog/v.0.3/sql-import/import-time-for-call.sql'
```
Comment (SergeyKoval) [inline src/main/resources/application.yaml:27]: Why are you removing some changes?

**Review by SergeyKoval — CHANGES_REQUESTED** [review]

**Comment by SergeyKoval** [comment]
Did you check which SQL queries appears on your update?

@ToString.Exclude
@OneToMany(fetch = FetchType.LAZY, cascade = CascadeType.ALL)
@JoinColumn(name = "fm_id")
private List<TimeForCall> timeForCallList;

I'm expecting some time for call auto changes...

File: src/main/resources/application.yaml (hunk 1 above)
Comment (ShirshovAlex) [reply src/main/resources/application.yaml:27]: it was a bad from previus PR

File: src/main/java/by/exadel/internship/controller/FormController.java [hunk 2]
```diff
@@ -55,4 +56,10 @@ public void restoreForm(@PathVariable UUID formId) {
         formService.restoreFormById(formId);
     }
 
+    @PutMapping("/updateStatus")
```
Comment (SergeyKoval) [inline src/main/java/by/exadel/internship/controller/FormController.java:59]: Shouldn't this endpoint be available only for users with some permissions?

**Review by SergeyKoval — CHANGES_REQUESTED** [review]

File: src/main/java/by/exadel/internship/controller/FormController.java (hunk 2 above)
Comment (ShirshovAlex) [reply src/main/java/by/exadel/internship/controller/FormController.java:59]: done

**Review by SergeyKoval — APPROVED** [review]

**Review by Maksim-maevsky — APPROVED** [review]

**Pull request merged**